lib/dark_worlds_server/bot/bot_client.ex

```diff
@@ -33,6 +33,10 @@ defmodule DarkWorldsServer.Bot.BotClient do
     {:reply, {:binary, Communication.player_use_skill(skill, angle)}, state}
   end
 
+  def handle_info({:use_inventory, inventory_at}, state) do
```

Comment:
Bot player are not sending this so they are not using items

Reply:
Yeah the bots are kind of broken since the loots behave differently from before, i think we could introduce another case to their decision list in another pr